_includes/android/getting-started.md

```diff
@@ -58,33 +25,19 @@ public class App extends Application {
     super.onCreate();
     Parse.initialize(new Parse.Configuration.Builder(this)
       .applicationId("YOUR_APP_ID")
+      .clientKey("YOUR_CLIENT_KEY")
       .server("http://localhost:1337/parse/")
       .build()
     );
   }
```

Comment:
Don't we have to do `ParseFCM.register(this);` in the Application instance if we want push?

Reply:
Nope, it is not required for FCM. I think we will leave the GCM instructions out of here, only mentioning it at the bottom, so that people will move away from it.

Reply:
I was playing with it last night but couldnt get stuff to fire unless I did the register command with FCM...you sure it works without it?

Reply:
Never mind, I was putting a breakpoint on the wrong place.  Works fine!

Reply:
See my updates about how a JobService is scheduled to run when the app starts:

https://github.com/rogerhu/Parse-SDK-Android/blob/8ec18723866d94f2ab15a3933f6c77fee1e5050f/gcm/README.md